### PR TITLE
(WIP)(MODULES-2656) Update Tests for BitVise Templates

### PR DESCRIPTION
**Pull request by cowofevil**
Update tests that are affected by the switch to BitVise Windows templates.
This required using PowerShell instead of Cygwin commands.

DO NOT MERGE! This PR is currently blocked by Beaker bugs.

### CODE REVIEW

File: tests/acceptance/pre-suite/00_install_certs.rb [hunk 1]
```diff
@@ -1,6 +1,9 @@
+#require 'pry-debugger'
```
Comment (cowofevil) [inline tests/acceptance/pre-suite/00_install_certs.rb]: Remove this!

File: tests/acceptance/pre-suite/00_install_certs.rb [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
 test_name "Install CA Certs"
 confine(:to, :platform => 'windows')
 
+temp_dir_name = 'windows\temp'
```
Comment (Iristyle) [inline tests/acceptance/pre-suite/00_install_certs.rb]: This is a bit hacky... trying to find a better way that isn't as hard-coded.

File: tests/test_run_scripts/integration_tests.sh [hunk 3]
```diff
@@ -10,7 +10,7 @@ declare -a ARGS
 # Argument Parsing
 if [ $# -eq 0 ]; then
   ARGS[0]='windows-2012r2-64mda'
-  ARGS[1]='http://neptune.puppetlabs.lan/2015.2/preview'
+  ARGS[1]='http://neptune.puppetlabs.lan/2015.2/ci-ready'
```
Comment (Iristyle) [inline tests/test_run_scripts/integration_tests.sh]: This looks unrelated to the host.

File: tests/manifests/user_scenarios/kitchen_sink.pp.erb [hunk 4]
```diff
@@ -51,6 +51,7 @@ dsc_package { 'install_7zip':
   dsc_productid => '',
   dsc_arguments => "/S /D=${seven_zip_install_path}"
 }
+->
```
Comment (Iristyle) [inline tests/manifests/user_scenarios/kitchen_sink.pp.erb]: This looks unrelated to the host... seems like a candidate for a separate maint PR

**Comment by glennsarti** [comment]
@cowofevil Can I close this PR?

**Comment by cowofevil** [comment]
@glennsarti Yes. Burn it with fire.

**Comment by glennsarti** [comment]
Closed with :fire:
